### ISSUE TITLE
Address table in intro lesson has inconsistent data columns

### ISSUE THREAD

**Issue by DanielBaird** [data-lessons/library-openrefine#26]
The table of addresses in `01-introduction` has an initial column containing addresses all concatenated into a single field, and following columns which I assume are there to show a nice clean breakdown of the address into separate fields.

However the following columns are erratically filled -- there are postcodes in the country column, etc.

I'm delivering this lesson soon so I might get some time to pull-req a fix for this.  If anyone's enthusiastic about it though, feel free to get ahead of me!

**Comment by drjwbaker** [comment]
@ostephens @pitviper6 @ccronje Any thoughts on this?

@DanielBaird Where are you delivering this lesson, may I ask?

**Comment by ostephens** [comment]
That was my error. I hate doing tables in Markdown!

Now corrected

**Comment by drjwbaker** [comment]
Thanks @ostephens!

**Comment by pitviper6** [comment]
Hi!  Sorry I've been so quiet lately, but we just finished up our LC workshop (very successful, we've got several people starting to really dig in to Open Refine). I'm also moving from San Diego to Chicago at the end of the week, so my life is super chaotic right now!  I'll be on the road until August 8, but after that, the San Diego team is planning on using August to work on the current lessons and adding exercises etc. based on or workshop experience.  (I'm moving but will still be working for UCSD through the end of the year).
